cinema/tests/test_movie_and_movie_session.py

```diff
@@ -0,0 +1,286 @@
+        movie_response_drama = self.client.get(
+            MOVIE_LIST_URL, {"genres": f"{self.genre_drama.id}"}
+        )
+        movie_response_something = self.client.get(
+            MOVIE_LIST_URL, {"genres": f"89982"}
+        )
+
+        movie_serializer_horror = MovieListSerializer(self.movie_apalon, many=False)
+        movie_serializer_drama = MovieListSerializer(self.movie_vinston, many=False)
+
+        self.assertEqual(
+            movie_serializer_horror.data["genres"],
+            movie_response_horror.data[0]["genres"],
+        )
+        self.assertEqual(
+            movie_serializer_drama.data["genres"],
+            movie_response_drama.data[0]["genres"],
+        )
+        self.assertNotEqual(
+            movie_serializer_drama.data["genres"],
+            movie_response_horror.data[0]["genres"],
+        )
+        self.assertEqual(movie_response_something.status_code, status.HTTP_200_OK)
+        self.assertEqual(movie_response_something.data, [])
+
+    def test_filter_movie_session_by_date(self):
+        movie_session_response_2023 = self.client.get(
+            MOVIE_SESSION_LIST_URL,
+            {
+                "date": f"{self.movie_session_2023.show_time.year}-"
+                f"{self.movie_session_2023.show_time.month}-"
+                f"{self.movie_session_2023.show_time.day}"
+            },
+        )
+
+        movie_session_response_2024 = self.client.get(
+            MOVIE_SESSION_LIST_URL,
+            {
+                "date": f"{self.movie_session_2024.show_time.year}-"
+                f"{self.movie_session_2024.show_time.month}-"
+                f"{self.movie_session_2024.show_time.day}"
+            },
+        )
+        movie_session_response_1762 = self.client.get(
+            MOVIE_SESSION_LIST_URL,
+            {
+                "date": "1762-01-01",
+            },
+        )
+
+        movie_session_serializer_2023 = MovieSessionListSerializer(
+            self.movie_session_2023, many=False
+        )
+
+        movie_session_serializer_2024 = MovieSessionListSerializer(
+            self.movie_session_2024, many=False
+        )
+
+        self.assertEqual(
+            movie_session_response_2023.data[0]["show_time"],
+            movie_session_serializer_2023.data["show_time"],
+        )
+
+        self.assertEqual(
+            movie_session_response_2024.data[0]["show_time"],
+            movie_session_serializer_2024.data["show_time"],
+        )
+
+        self.assertNotEqual(
+            movie_session_response_2024.data[0]["show_time"],
+            movie_session_serializer_2023.data["show_time"],
+        )
+
+        self.assertEqual(movie_session_response_1762.status_code, status.HTTP_200_OK)
+        self.assertEqual(movie_session_response_1762.data, [])
+
+    def test_filter_movie_session_by_movie_title(self):
+        movie_session_response_vinston = self.client.get(
+            MOVIE_SESSION_LIST_URL, {"movie": f"{self.movie_session_2023.movie.id}"}
```

Comment:
You're passing the movie ID as a string to the filter. While this is not wrong, it would be more appropriate to pass it as an integer, since IDs are typically integers.